andela/krypton-ah-frontend
# #164138899 Fix network error response issue

#### What does this PR do?
Fix network error response issue
#### Description of Task to be completed?
#### How should this be manually tested?
* Git clone the repo
* cd into the directory
* run npm run dev
* open the redux dev tools from the browser and dispatch NETWORK_ERROR action
* You will see the changes in the state
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/164138899
#### Screenshots (if appropriate)
<img width="1431" alt="screenshot 2019-02-21 at 4 03 45 pm" src="https://user-images.githubusercontent.com/20207295/53180337-ac91b280-35f5-11e9-9d8c-047217399656.png">

#### Questions:
